docs/assets/includes/nav-multistep.md

```diff
@@ -0,0 +1,20 @@
+<nav class="nav-multistep">
```

Comment:
I think we should experiment with including a small introductory sentence before the navigation, for example:

```suggestion
This page is step {{ page.meta.onboarding_step }} of the following guided path:

<nav class="nav-multistep">
```

Or a simpler version that doesn't mention the step:

```suggestion
This page is part of the following guided path:

<nav class="nav-multistep">
```

Reply:
Agreed. Done 👍